inst/scripts/SEL3D.R

```diff
@@ -0,0 +1,139 @@
+#Pseudocode may look like actual code...please ignore...
+getPdbIds <- function(targetSeq, startPos = 1 , endPos = nchar(targetSeq), convert = TRUE, silent = TRUE){
+  
+  #get packages
+
+  library(bio3d)
+  library(RCurl)
+  library(XML)
+  library(seqinr)
+  
```

Comment:
You can look at Boris's STRING.R file linked below or other merged files in the directory R for examples

https://github.com/hyginn/rete/blob/master/R/importNet.STRING.R